Added a feedback section in the website: Resolved issue #164

Prahmodh-R opened the pull request:
This is my PR to issue #164.
I've added a feedback section in the landing page of the website.
Mail-Id: pramodhr181@gmail.com
Devfolio Username: Prahmodh_Raj
<img width="958" alt="image" src="https://user-images.githubusercontent.com/89332177/190858904-a75aa1bf-b343-4d9c-8d82-e81d9f55bf58.png">
![image](https://user-images.githubusercontent.com/89332177/190858926-ff808f79-cf31-456f-9af0-c5210f10cf86.png)

Prahmodh-R commented:
@narayan954 Could you please verify and merge this PR for issue #164 . I've resolved  the merge branch conflicts

narayan954 commented:
hey @Prahmodh-R can you remove those pictures, ig it'd look better without a picture

Prahmodh-R commented:
@narayan954 So shall I just have the names of the people without their pictures?

narayan954 commented:
yes @Prahmodh-R

Prahmodh-R commented:
I've removed the pictures and added their designations instead, is that fine? @narayan954 
![image](https://user-images.githubusercontent.com/89332177/190890517-abde60e0-5264-4dae-8ca2-559d317aed40.png)

narayan954 reviewed (approved):
looks great! 
Ps. I removed the underline as it didn't follow other pages styling.

narayan954 commented:
Thanks for your contribution! :)

Prahmodh-R commented:
@narayan954 Could you pls add the SSOC tag to this PR too, thank you!

narayan954 commented:
done :)
